deploy/helms/dex/templates/deployment.yaml

```diff
@@ -34,7 +34,6 @@ spec:
       containers:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
-          imagePullPolicy: {{ .Values.image.pullPolicy }}
```

Comment:
Changing this value doesn't have effect in fact but will require dex component new release since it's been shipped in a separate pipeline. Let's skip it for now since it doesn't worth efforts as for me.